lib/src/components/NxCombobox/types.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as PropTypes from 'prop-types';
+import { omit } from 'ramda';
+import { HTMLAttributes, ReactNode } from 'react';
+import { PublicProps as NxTextInputProps, propTypes as nxTextInputPropTypes } from '../NxTextInput/types';
+
+import DataItem from '../../util/DataItem';
+
+type InputProps = Omit<NxTextInputProps, 'type' | 'value' | 'isPristine' | 'readOnly'>;
+
+export interface Props<T extends string | number = string> extends Omit<HTMLAttributes<HTMLDivElement>, 'onSelect'> {
+  searchText: string;
+  loading?: boolean | null;
+  error?: ReactNode;
+  matches: DataItem<T>[];
+  onSelect: (m: DataItem<T>) => void;
+  onSearch: (s: string) => void;
+  onSearchTextChange: (s: string) => void;
+  long?: boolean | null;
+  short? : boolean | null;
```

Comment:
This should be handled the same way that it is on `NxTextInput` - by just allowing the caller to specify some modifier classes

Reply:
Updated here dd696a514303bcca92c57ac4fa8da09a7fe4fdd2.